Content/Items/Gear/Gear.cs

```diff
@@ -13,8 +14,21 @@ internal abstract class Gear : PoTItem
 {
 	protected virtual string GearLocalizationCategory => GetType().Name;
 
-	private Socketable[] _sockets = []; // [new Imps()];
+	private Socketable[] _sockets = [];
 	private int _selectedSocket;
+	
+	public override void OnCreated(ItemCreationContext context)
+	{
+		base.OnCreated(context);

```

Comment:
Note that most tMod (NOT vanilla) methods - specifically, those in a ModLoader namespace - do nothing in `base` aside from return a value if necessary. You don't have to delete this and it's not bad practice, but I feel it's good to know.